docs/using/api.ipynb

```diff
@@ -541,218 +543,216 @@
+    "cache.cache_notebook_bundle(nb_bundle)"
+   ],
    "outputs": [
     {
+     "output_type": "execute_result",
      "data": {
       "text/plain": [
        "NbCacheRecord(pk=2)"
       ]
      },
-     "execution_count": 18,
      "metadata": {},
-     "output_type": "execute_result"
+     "execution_count": 18
     }
    ],
-   "source": [
-    "nb_bundle = NbBundleIn(\n",
-    "    nb=notebook,\n",
-    "    uri=Path(\"example_nbs\", \"basic.ipynb\"),\n",
-    "    data={\"tag\": \"mytag\"}\n",
-    ")\n",
-    "cache.cache_notebook_bundle(nb_bundle)"
-   ]
+   "metadata": {}
   },
   {
    "cell_type": "code",
    "execution_count": 19,
-   "metadata": {},
+   "source": [
+    "print(tabulate_cache_records(\n",
+    "    cache.list_cache_records(), path_length=1, hashkeys=True\n",
+    "))"
+   ],
    "outputs": [
     {
-     "name": "stdout",
      "output_type": "stream",
+     "name": "stdout",
      "text": [
       "  ID  Origin URI    Created           Accessed          Hashkey\n",
       "----  ------------  ----------------  ----------------  --------------------------------\n",
-      "   2  basic.ipynb   2020-03-13 14:21  2020-03-13 14:21  74933d8a93d1df9caad87b2e6efcdc69\n",
-      "   1  basic.ipynb   2020-03-13 14:21  2020-03-13 14:21  818f3412b998fcf4fe9ca3cca11a3fc3\n"
+      "   2  basic.ipynb   2021-08-04 02:25  2021-08-04 02:25  07e6a47c8c180cb7851ede6dbb088769\n",
+      "   1  basic.ipynb   2021-08-04 02:23  2021-08-04 02:25  94c17138f782c75df59e989fffa64e3a\n"
      ]
     }
    ],
-   "source": [
-    "print(tabulate_cache_records(\n",
-    "    cache.list_cache_records(), path_length=1, hashkeys=True\n",
-    "))"
-   ]
+   "metadata": {}
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
    "source": [
     "Notebooks are retained in the cache, until the cache limit is reached,\n",
     "at which point the oldest notebooks are removed."
-   ]
+   ],
+   "metadata": {}
   },
   {
    "cell_type": "code",
    "execution_count": 20,
-   "metadata": {},
+   "source": [
+    "cache.get_cache_limit()"
+   ],
    "outputs": [
     {
+     "output_type": "execute_result",
      "data": {
       "text/plain": [
        "1000"
       ]
      },
-     "execution_count": 20,
      "metadata": {},
-     "output_type": "execute_result"
+     "execution_count": 20
     }
    ],
-   "source": [
-    "cache.get_cache_limit()"
-   ]
+   "metadata": {}
   },
   {
    "cell_type": "code",
    "execution_count": 21,
-   "metadata": {},
-   "outputs": [],
    "source": [
     "cache.change_cache_limit(100)"
-   ]
+   ],
+   "outputs": [],
+   "metadata": {}
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
    "source": [
-    "(use/api/stage)=\n",
+    "(use/api/project)=\n",
     "\n",
     "## Staging Notebooks for Execution"
-   ]
+   ],
+   "metadata": {}
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
    "source": [
     "Notebooks can be staged, by adding the path as a stage record.\n",
```

Comment:
Should this be renamed something like `## Add notebooks to a project for execution`?

And in general this section uses "staging", "the staged notebook" etc, rather than "project" terminology, which is a bit confusing as I'm not sure how "staging" and "project" relate to one another